ee/billing/quota_limiting.py

```diff
@@ -80,6 +82,11 @@ def org_quota_limited_until(organization: Organization, resource: QuotaResource)
     is_quota_limited = usage + todays_usage >= limit + OVERAGE_BUFFER[resource]
     billing_period_end = round(dateutil.parser.isoparse(organization.usage["period"][1]).timestamp())
 
+    if is_quota_limited and organization.never_drop_data:
+        if is_email_available():
+            send_over_quota_but_not_dropped_email_to_cs.delay(organization.id)
+        return None
```

Comment:
This will result in cs getting an email every hour or so. Is this what we want?

Reply:
Oh ha definitely not, I thought about that but then forgot when I put the PR up, thank you!